cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -282,6 +282,13 @@ func (p *BicepProvider) Destroy(
 				return
 			}
 
+			asyncContext.SetProgress(&DestroyProgress{Message: "Getting API Managements to purge", Timestamp: time.Now()})
```

Comment:
Maybe "Getting API Management Services to purge"?   I think Azure calls these an instance of the resource a "API Management Service", and it feels like it reads a little better.